src/driver/transaction.rs

```diff
@@ -118,6 +122,10 @@ impl RustTransaction {
             querystring.push_str(format!(" {}", &read_var.to_str_option()).as_str());
         }
 
+        if let Some(_) = self.deferable {
```

Comment:
I think it's better to use the default `if condition`, `if let` usually used in other situations:

```
if self.deferable.is_some() {
    querystring.push_str("SET CONSTRAINTS ALL DEFERRED");
}
```